airbyte-integrations/connectors/source-stripe/source_stripe/schemas/invoices.json

```diff
@@ -103,7 +103,7 @@
               "type": ["null", "integer"]
             },
             "redeem_by": {
-              "type": ["null", "number"]
+              "type": ["null", "integer"]
```

Comment:
I believe this is the only breaking change here? do we need to move it to integer?

Reply:
I believe yes, this is timestamp
https://stripe.com/docs/api/coupons/object#coupon_object-redeem_by